src/weather_man.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+const rootUrl = 'http://api.openweathermap.org/data/2.5/weather?q=';
+const APIKey = '&APPID=81ebd306a97bf6e5342257562b958514';
+
+class WeatherMan extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {city: "",
```

Comment:
It's kindof just picky formatting, but I think it's more readable to format your hash literals with each key/value pair on a new line (including the first pair). Also adding a comma at the end of the last one lets you reorder them easily and you won't get compile errors if you delete the first one.
```
constructor(props {
    super(props)
    this.state = {
        city: "",
        weather: "",
    }
}
```
I would use the same formatting down on line `19` for your `setState` parameters too.